protobufs/gen/swagger/fuuu/entity.swagger.json

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
What is this?? `fuuu`

Reply:
:) leftover from trying out stuff. I will remove it. good catch